src/web/src/features/sign-up/index.js

```diff
@@ -0,0 +1,154 @@
+
+  const handleSignUp = (values, setSubmitting) => {
+    setSubmitting(false);
+    (async () => {
+      try {
+        const {
+          data: { message }
+        } = await axiosInstance.post(`/user/create`, {
+          username: values.username,
+          email: values.email,
+          password: values.password
+        });
+        console.log(message);
+
+        const {
+          data: {
+            data: { accessToken, uid }
+          }
+        } = await axiosInstance.post(`/user/login`, {
+          email: values.email,
+          password: values.password
+        });
+
+        axiosInstance.defaults.headers.common['Authorization'] = `Bearer ${accessToken}`;
+
+        const {
+          data: { data }
+        } = await axiosInstance.get(`/user/${uid}`);
+        localStorage.setItem('SW_accessToken', accessToken);
+        localStorage.setItem('SW_uid', uid);
+        setLoggedInUser(data);
+      } catch (err) {
+        console.error(err);
+      }
+    })();
+  };
+
+  // TODO: Create a non-expiry token to use instead of having credentials here
+  return (
+    <>
+      <Formik
+        initialValues={{
+          email: '',
+          username: '',
+          password: '',
+          confirmPassword: ''
+        }}
+        validationSchema={SignUpSchema}
+        onSubmit={(values, { setSubmitting }) => handleSignUp(values, setSubmitting)}
+      >
+        {({ values, handleChange }) => (
+          <Form>
+            <Paper className={classes.logInPaper}>
+              <Typography variant="h4">Sign up</Typography>
+
+              <TextField
+                name="email"
+                value={values.email}
+                type="email"
+                placeholder="jdoe@email.com"
+                onChange={handleChange}
+              ></TextField>
+              <ErrorMessage
+                name="email"
+                render={(msg) => <div style={{ color: 'red' }}>{msg}</div>}
+              />
+
+              <TextField
+                name="username"
+                value={values.username}
+                type="text"
+                placeholder="JohnDoe123"
+                onChange={handleChange}
+              ></TextField>
+              <ErrorMessage
+                name="username"
+                render={(msg) => <div style={{ color: 'red' }}>{msg}</div>}
+              />
+
+              <TextField
+                name="password"
+                value={values.password}
+                type="password"
+                placeholder="******"
+                onChange={handleChange}
+              ></TextField>
+              <ErrorMessage
+                name="password"
+                render={(msg) => <div style={{ color: 'red' }}>{msg}</div>}
+              />
+
+              <TextField
+                name="confirmPassword"
+                value={values.confirmPassword}
+                type="password"
+                placeholder="******"
+                onChange={handleChange}
```

Comment:
add `label="Confirm password"` as props for `TextField`

Reply:
Got it, I'll add the labels for all the fields